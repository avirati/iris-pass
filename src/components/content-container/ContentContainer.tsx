import React from 'react'

import { Flex } from 'shared-components'

interface IContentContainer {
  children?: React.ReactNode
}

export const ContentContainer: React.FC<IContentContainer> = ({ children }) => {
  return (
    <Flex
      css={{
        p: '$5'
      }}
    >
      {children}
    </Flex>
  )
}