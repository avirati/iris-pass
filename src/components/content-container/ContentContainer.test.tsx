import { render } from '@testing-library/react'

import { ContentContainer } from './ContentContainer'

describe('ContentContainer Component', () => {
  it('renders', () => {
    const { container } = render(<ContentContainer>Content</ContentContainer>)
    expect(container).toMatchSnapshot()
  })

})