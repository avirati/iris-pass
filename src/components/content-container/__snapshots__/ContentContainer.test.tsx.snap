// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentContainer Component renders 1`] = `
<div>
  <div
    class="c-dhzjXW c-dhzjXW-ijcowMO-css"
  >
    Content
  </div>
</div>
`;
